transforms/.make.workflows

```diff
@@ -13,16 +13,30 @@ define set_env_var
 	$(eval export $(1)=$(2))
 endef
 
-# FIXME
+ifeq ($(KFPv2), 1)
```

Comment:
we have the same code snippet in `kfp_ray_components/Makefile`, can they be combined ?

Reply:
Fixed. Thanks